### PR TITLE
Added hover effects to patreon and paypal

### PR DESCRIPTION
**Pull request by Ananthavijay**
Fixes https://github.com/CircuitVerse/CircuitVerse/issues/1048

#### Describe the changes you have made in this PR -

Added hover effects to patreon and paypal

### Screenshots of the changes (If any) -
CircuitVerse - Support the platform

### CODE REVIEW

**Comment by Ananthavijay** [comment]
@nitin10s Please review

File: app/assets/stylesheets/logixRelease.scss [hunk 1]
```diff
@@ -242,6 +242,11 @@ a.btn.btn-outline-danger {
   border-radius: 8px;
   padding: 4px;
   box-shadow: 3px 3px 6px $shadow-black;
+  transition: box-shadow .2s ease-in;
+}
+
+.patreon:hover {
```
Comment (tachyons) [inline app/assets/stylesheets/logixRelease.scss]: Move to about.scss, also prefer sass pseudo class syntax

Comment (Ananthavijay) [reply app/assets/stylesheets/logixRelease.scss]: This is not in about page, this is in contribute page

File: app/assets/stylesheets/logixRelease.scss [hunk 2]
```diff
@@ -242,6 +242,10 @@ a.btn.btn-outline-danger {
   border-radius: 8px;
   padding: 4px;
   box-shadow: 3px 3px 6px $shadow-black;
+  transition: box-shadow .2s ease-in;
+  &:hover {
```
[the remark this answers is not in the document]
Comment (Ananthavijay) [reply app/assets/stylesheets/logixRelease.scss]: Done

File: app/assets/stylesheets/logixRelease.scss [hunk 3]
```diff
@@ -242,6 +242,11 @@ a.btn.btn-outline-danger {
   border-radius: 8px;
   padding: 4px;
   box-shadow: 3px 3px 6px $shadow-black;
+  transition: box-shadow .2s ease-in;
+
```
[the remark this answers is not in the document]
Comment (Ananthavijay) [reply app/assets/stylesheets/logixRelease.scss]: Done

File: app/assets/stylesheets/logixRelease.scss (hunk 1 above)
Comment (tachyons) [reply app/assets/stylesheets/logixRelease.scss]: then contribute.scss

Comment (Ananthavijay) [reply app/assets/stylesheets/logixRelease.scss]: 👍

File: app/assets/stylesheets/contribute.scss [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+.cardimage-contribute{
+  width: 150px;
+  height: 150px;
+}
+
+.cardbody-contribute > h3{
+  text-align: center;
+}
```
[the remark this answers is not in the document]
Comment (Ananthavijay) [reply app/assets/stylesheets/contribute.scss]: Done

File: app/assets/stylesheets/contribute.scss [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  box-shadow: 3px 3px 6px $shadow-black;
+  transition: box-shadow .2s ease-in;
+
+  &:hover {
+    box-shadow: 0px 9px 9px $shadow-black;
+  }
+}
+
+.patreon > img 
+  width: 160px;
```
[the remark this answers is not in the document]
Comment (Ananthavijay) [reply app/assets/stylesheets/contribute.scss]: Done

**Comment by Ananthavijay** [comment]
I have transferred the styles related to contribute page to contribute.scss. @tachyons @nitin10s Please review

**Pull request merged**